torch/ao/quantization/fx/backend_config/native.py

```diff
@@ -92,16 +96,205 @@ def _get_default_op_backend_config(op, dtype_configs):
+        "reference_quantized_module_for_root": nnqr.Linear,
+        "qat_module": nnqat.Linear,
+    })
+    # linear qat module
+    linear_configs.append({
+        "pattern": nnqat.Linear,
+        "observation_type": observation_type,
+        "dtype_configs": dtype_configs,
+        "root_module": torch.nn.Linear,
+        "reference_quantized_module_for_root": nnqr.Linear,
+    })
+    # functional linear
+    linear_configs.append({
+        "pattern": torch.nn.functional.linear,
+        "observation_type": observation_type,
+        "dtype_configs": dtype_configs,
+    })
+    # linear relu, fused module
+    linear_configs.append({
+        "pattern": nni.LinearReLU,
+        "observation_type": observation_type,
+        "dtype_configs": dtype_configs ,
+        "root_module": torch.nn.Linear,
+        "reference_quantized_module_for_root": nnqr.Linear,
+        "qat_module": nniqat.LinearReLU,
+    })
+    # linear relu, qat fused module
+    linear_configs.append({
+        "pattern": nniqat.LinearReLU,
+        "observation_type": observation_type,
+        "dtype_configs": dtype_configs,
+        "root_module": torch.nn.Linear,
+        "reference_quantized_module_for_root": nnqr.Linear,
+    })
+    # linear relu, linear module + relu module
+    linear_configs.append({
+        "pattern": (torch.nn.ReLU, torch.nn.Linear),
+        "observation_type": observation_type,
+        "dtype_configs": dtype_configs,
+        "fuser_method": reverse_sequential_wrapper2(nni.LinearReLU),
+    })
+    # linear relu, linear module + functional relu
+    linear_configs.append({
+        "pattern": (F.relu, torch.nn.Linear),
+        "observation_type": observation_type,
+        "dtype_configs": dtype_configs,
+        "fuser_method": reverse_sequential_wrapper2(nni.LinearReLU),
+    })
+    # linear relu, functional linear + relu module
+    linear_configs.append({
+        "pattern": (torch.nn.ReLU, F.linear),
+        "observation_type": observation_type,
+        "dtype_configs": dtype_configs,
+    })
+    # linear relu, functional linear + functional relu
+    linear_configs.append({
+        "pattern": (F.relu, F.linear),
+        "observation_type": observation_type,
+        "dtype_configs": dtype_configs,
+    })
+    # linear bn, fused module
+    linear_configs.append({
+        "pattern": nni.LinearBn1d,
+        "observation_type": observation_type,
+        "dtype_configs": dtype_configs,
+        "root_module": torch.nn.Linear,
+        "reference_quantized_module_for_root": nnqr.Linear,
+        "qat_module": nniqat.LinearBn1d,
+    })
+    # linear bn, qat fused module
+    linear_configs.append({
+        "pattern": nniqat.LinearBn1d,
+        "observation_type": observation_type,
+        "dtype_configs": dtype_configs,
+        "root_module": torch.nn.Linear,
+        "reference_quantized_module_for_root": nnqr.Linear,
+    })
+    return linear_configs
+
+def _get_conv_configs():
```

Comment:
same for conv I think, we can classify these configs a bit, to make sure it is easier to add things in the future.